server/game/gameSystems/LookAtSystem.ts

```diff
@@ -1,77 +1,39 @@
 import { AbilityContext } from '../core/ability/AbilityContext';
-import { BaseCard } from '../core/card/BaseCard';
-import { EventName } from '../core/Constants';
-import { GameEvent } from '../core/event/GameEvent';
-import { CardTargetSystem, ICardTargetSystemProperties } from '../core/gameSystem/CardTargetSystem';
+import { EventName, ViewCardType } from '../core/Constants';
+import { ViewCardSystem, IViewCardProperties } from './ViewCardSystem';
 
-export interface ILookAtProperties extends ICardTargetSystemProperties {
-    message?: string | ((context) => string);
-    messageArgs?: (cards: any) => any[];
-}
+export type ILookAtProperties = Omit<IViewCardProperties, 'viewType'>;
 
-export class LookAtSystem extends CardTargetSystem<ILookAtProperties> {
+export class LookAtSystem extends ViewCardSystem {
     public override readonly name = 'lookAt';
     public override readonly eventName = EventName.OnLookAtCards;
```

Comment:
nit: if each event is only going to look at a single card at a time, then I think a more consistent naming for this is `OnLookAtCard` (singular)